### DISCUSSION TITLE
表分区的设计问题

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#13468] [A - General / Q&A]
1、Doris的表分区是什么样的，假设我一个3节点的BE集群，如果分区1个，是不是只会在一个节点上有数据？ 数据是怎么均衡的？好像找不到对应资料哦
2、对于即席查询的场景，有必要用动态分区吗？ 是否用地市分区、区域分桶即可？
3、针对统计的场景，才使用动态分区才合理呢？

**Comment by dataalive** [comment]
1. 这个数据分布首先是根据分区分桶到tablet粒度，然后在多少机器上取决于多少副本，比如你所说的场景，如果是3副本的话，那就会在3台节点上都会有这个分区的tablets。
2. 分区还是建议用的，尤其是你的查询里会把分区的字段作为条件的时候。
3. 动态分区只是能自动创建，比如按天之类的，这个如果是统计的场景，看下天级别的数据量，可能按天即可，如果比较小按周按月都ok
